api/projects.ts

```diff
@@ -351,10 +403,21 @@ export function migrateApp(
 
 export function checkMigrationStatus(
   accountId: number,
-  id: number
+  id: number,
+  targetPlatformVersion: string = PLATFORM_VERSIONS.v2023_2
 ): HubSpotPromise<PollAppResponse> {
+  // TODO: This is a WIP, we need to wait for the BE team to implement it for the new platform version
+  if (
+    targetPlatformVersion === PLATFORM_VERSIONS.unstable ||
+    targetPlatformVersion === PLATFORM_VERSIONS.v2025_2
+  ) {
+    return http.get<PollAppResponse>(accountId, {
+      url: `${MIGRATIONS_API_PATH_V2}/migrations/${id}`,
+    });
+  }
```

Comment:
Waiting on the BE team to implement this endpoint and then I will update the URL and the Response shape